test/e2e/latest/e2e_suite.go

```diff
@@ -0,0 +1,396 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package latest
+
+import (
+	"encoding/base64"
```

Comment:
Also, the v3 tests need to be added in : https://github.com/kubernetes-sigs/kubebuilder/blob/master/test_e2e.sh

Reply:
They were: https://github.com/kubernetes-sigs/kubebuilder/pull/1448/files#diff-12fa9d4d5a59e0c6b270872dc2f70ec5R46.

Reply:
You can close this one as solved.